dot/parachain/types/async_backing.go

```diff
@@ -17,3 +17,76 @@ type AsyncBackingParams struct {
 	// When async backing is disabled, the only valid value is 0.
 	AllowedAncestryLen uint32 `scale:"2"`
 }
+
+// InboundHrmpLimitations constraints on inbound HRMP channels.
+type InboundHrmpLimitations struct {
```

Comment:
```suggestion
type InboundHRMPLimitations struct {
```
go best practice is to capitalize abbreviations.